lib/gitlab/satellite/files/file_action.rb

```diff
@@ -14,10 +14,11 @@ def safe_path?(path)
       end
 
       def write_file(abs_file_path, content, file_encoding = 'text')
-        if file_encoding == 'base64'
-          File.open(abs_file_path, 'wb') { |f| f.write(Base64.decode64(content)) }
-        else
-          File.open(abs_file_path, 'w') { |f| f.write(content) }
+		if file_encoding == 'base64'
```

Comment:
Use 2 (not -4) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.